Tasks/Common/kubernetes-common-v2/helmutility.ts

```diff
@@ -66,16 +67,24 @@ function getHelmDownloadURL(version: string): string {
 
 export async function getStableHelmVersion(): Promise<string> {
     try {
-        const downloadPath = await toolLib.downloadTool(helmLatestReleaseUrl);
-        const response = JSON.parse(fs.readFileSync(downloadPath, 'utf8').toString().trim());
-        if (!response.tag_name)
-        {
-            return stableHelmVersion;
-        }
-        
-        return response.tag_name;
+        const downloadPath = await toolLib.downloadTool(helmAllReleasesUrl);
+        const responseArray = JSON.parse(fs.readFileSync(downloadPath, 'utf8').toString().trim());
+        let latestHelmVersion = semver.clean(stableHelmVersion);
+        responseArray.forEach(response => {
+            if (response && response.tag_name) {
+                let currentHelmVerison = semver.clean(response.tag_name.toString());
```

Comment:
Does this not break when `rc` is found in the version?

Reply:
No, it just removes the 'v' in the beginning of the version. So, from what I tested, 'v3.1.0-rc.3' would be changed to '3.1.0-rc.3'.